color/src/tag.rs

```diff
@@ -48,6 +48,8 @@ pub enum ColorSpaceTag {
     ProphotoRgb = 10,
     /// The [`Rec2020`] color space.
     Rec2020 = 11,
+    /// The [`Aces2065_1`] color space.
+    Aces2065_1 = 15,
     /// The [`AcesCg`] color space.
     AcesCg = 12,
     /// The [`XyzD50`] color space.
```

Comment:
How do we want to keep this sorted? Renumbering may be breaking.

Reply:
I'm okay with what you did, but I think probably would be good to put a comment somewhere in there about it. I don't think re-numbering is good except (maybe) at breaking changes.

Reply:
I tend to agree that re-numbering in a patch release would be bad. But I also think that we should renumber these when we next make a breaking change. If people are storing color spaces in a file, then I'd like to encourage them to make their own ordering.